.github/workflows/performance.yml

```diff
@@ -44,6 +44,10 @@ jobs:
         uses: ./.github/actions/make_init
       - name: Install playwright
         run: python -m playwright install --with-deps
+      - name: Install integration dependencies
+        run: |
+          source venv/bin/activate
+          uv pip install polars
```

Comment:
We install here only `polars`, and not `integration-requirements.txt` because `ray` (another integration requirement is not supported by Python 3.13).